### ISSUE TITLE
Error in Ladder Board

### ISSUE THREAD

**Issue by axel1200** [FAForever/downlords-faf-client#621]
``` 
Exception in thread "JavaFX Application Thread" java.lang.NullPointerException
	at com.faforever.client.rankedmatch.Ladder1v1Controller.lambda$addNodeOnTopOfBar$17(Ladder1v1Controller.java:300)
	at com.sun.javafx.binding.ExpressionHelper$SingleChange.fireValueChangedEvent(ExpressionHelper.java:182)
	at com.sun.javafx.binding.ExpressionHelper.fireValueChangedEvent(ExpressionHelper.java:81)
	at javafx.beans.property.ReadOnlyObjectPropertyBase.fireValueChangedEvent(ReadOnlyObjectPropertyBase.java:74)
	at javafx.beans.property.ReadOnlyObjectWrapper.fireValueChangedEvent(ReadOnlyObjectWrapper.java:102)
	at javafx.beans.property.ObjectPropertyBase.markInvalid(ObjectPropertyBase.java:112)
	at javafx.beans.property.ObjectPropertyBase.set(ObjectPropertyBase.java:146)
	at javafx.scene.Node.setParent(Node.java:720)
	at javafx.scene.Parent$2.onProposedChange(Parent.java:497)
	at com.sun.javafx.collections.VetoableListDecorator.remove(VetoableListDecorator.java:329)
	at com.sun.javafx.collections.VetoableListDecorator.remove(VetoableListDecorator.java:221)
	at javafx.scene.chart.BarChart.seriesRemoved(BarChart.java:332)
	at javafx.scene.chart.XYChart.lambda$new$550(XYChart.java:139)
	at com.sun.javafx.collections.ListListenerHelper$SingleChange.fireValueChangedEvent(ListListenerHelper.java:164)
	at com.sun.javafx.collections.ListListenerHelper.fireValueChangedEvent(ListListenerHelper.java:73)
	at javafx.collections.ObservableListBase.fireChange(ObservableListBase.java:233)
	at javafx.collections.ListChangeBuilder.commit(ListChangeBuilder.java:482)
	at javafx.collections.ListChangeBuilder.endChange(ListChangeBuilder.java:541)
	at javafx.collections.ObservableListBase.endChange(ObservableListBase.java:205)
	at javafx.collections.ModifiableObservableListBase.setAll(ModifiableObservableListBase.java:90)
	at javafx.collections.ObservableListBase.setAll(ObservableListBase.java:250)
	at com.faforever.client.rankedmatch.Ladder1v1Controller.lambda$plotRatingDistributions$16(Ladder1v1Controller.java:293)
	at com.sun.javafx.application.PlatformImpl.lambda$null$172(PlatformImpl.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$173(PlatformImpl.java:294)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at com.sun.glass.ui.win.WinApplication.lambda$null$147(WinApplication.java:177)
	at java.lang.Thread.run(Thread.java:748)
Exception in thread "JavaFX Application Thread" java.lang.NullPointerException
	at com.faforever.client.rankedmatch.Ladder1v1Controller.lambda$addNodeOnTopOfBar$17(Ladder1v1Controller.java:300)
	at com.sun.javafx.binding.ExpressionHelper$SingleChange.fireValueChangedEvent(ExpressionHelper.java:182)
	at com.sun.javafx.binding.ExpressionHelper.fireValueChangedEvent(ExpressionHelper.java:81)
	at javafx.beans.property.ReadOnlyObjectPropertyBase.fireValueChangedEvent(ReadOnlyObjectPropertyBase.java:74)
	at javafx.beans.property.ReadOnlyObjectWrapper.fireValueChangedEvent(ReadOnlyObjectWrapper.java:102)
	at javafx.beans.property.ObjectPropertyBase.markInvalid(ObjectPropertyBase.java:112)
	at javafx.beans.property.ObjectPropertyBase.set(ObjectPropertyBase.java:146)
	at javafx.scene.Node.setParent(Node.java:720)
	at javafx.scene.Parent$2.onProposedChange(Parent.java:497)
	at com.sun.javafx.collections.VetoableListDecorator.remove(VetoableListDecorator.java:329)
	at com.sun.javafx.collections.VetoableListDecorator.remove(VetoableListDecorator.java:221)
	at javafx.scene.chart.BarChart.seriesRemoved(BarChart.java:332)
	at javafx.scene.chart.XYChart.lambda$new$550(XYChart.java:139)
	at com.sun.javafx.collections.ListListenerHelper$SingleChange.fireValueChangedEvent(ListListenerHelper.java:164)
	at com.sun.javafx.collections.ListListenerHelper.fireValueChangedEvent(ListListenerHelper.java:73)
	at javafx.collections.ObservableListBase.fireChange(ObservableListBase.java:233)
	at javafx.collections.ListChangeBuilder.commit(ListChangeBuilder.java:482)
	at javafx.collections.ListChangeBuilder.endChange(ListChangeBuilder.java:541)
	at javafx.collections.ObservableListBase.endChange(ObservableListBase.java:205)
	at javafx.collections.ModifiableObservableListBase.setAll(ModifiableObservableListBase.java:90)
	at javafx.collections.ObservableListBase.setAll(ObservableListBase.java:250)
	at com.faforever.client.rankedmatch.Ladder1v1Controller.lambda$plotRatingDistributions$16(Ladder1v1Controller.java:293)
	at com.sun.javafx.application.PlatformImpl.lambda$null$172(PlatformImpl.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$173(PlatformImpl.java:294)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at com.sun.glass.ui.win.WinApplication.lambda$null$147(WinApplication.java:177)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by micheljung** [comment]
How to reproduce?

**Comment by micheljung** [comment]
This has been fixed in db7fa45
